website/core/HeaderLinks.js

```diff
@@ -16,8 +16,8 @@ var HeaderLinks = React.createClass({
   linksInternal: [
     {section: 'docs', href: 'docs/getting-started.html', text: 'Docs'},
     {section: 'support', href: 'support.html', text: 'Support'},
-    {section: 'newsletter', href: 'http://reactnative.cc', text: 'Newsletter'},
     {section: 'showcase', href: 'showcase.html', text: 'Showcase'},
+    {section: 'blog', href: 'blog/', text: 'Blog'},
```

Comment:
@vjeux this should be `href: '/react-native/blog'`.
Otherwise BASE setting will prepend `/react-native/release/XX` to the link and it won't work for people who navigate other versions but stable.